server/dune

```diff
@@ -0,0 +1,11 @@
+(library
+ (name dhcp_server)
+ (public_name charrua-server)
+ (private_modules dhcp_lexer dhcp_parser ast util)
```

Comment:
exposing an unwrapped module called Ast and Util seems like a bit of a disaster. Wouldnt this cause a module clash even through its private?

Reply:
done in 6313cae